mcan/src/bus.rs

```diff
@@ -157,6 +157,14 @@ pub trait DynAux {
     /// disable CAN operation.
     fn initialization_mode(&self);
 
+    /// Sets MCAN up for being shut down.
+    /// See `Power Down (sleep mode)`, in the peripheral docs, for user
+    /// consideration regarding clocking and message handling.
+    fn shutdown(&self);
+
+    /// Check if csa bit is set
+    fn ready_for_shutdown(&self) -> bool;
```

Comment:
Same here, `ready_for_powerdown` or `power_down_mode`?

Reply:
Agree. Think `ready_for_power_down` is fine, or `ready_for_sleep_mode` in case we're going with `sleep mode` over `power down mode`.

Reply:
I renamed it  to `is_ready_for_power_off` to make the difference between "power down mode" and "power off" even clearer.

What do you think?